### DISCUSSION TITLE
Issue intstalling Omada Controller LXC

### DISCUSSION THREAD

**Discussion by thealphabeta** [tteck/Proxmox#2444] [General]
Issue when installing - tried on 2 different proxmox installs
using:
default settings
proxmox shell 
proxmox 8.1.3

`[ERROR] in line 33: exit code 0: while executing command wget -qL http://security.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2.20_amd64.deb`

complete output:
  
 
Using Default Settings
Using Distribution: debian
Using debian Version: 12
Using Container Type: 0
Using Root Password: Automatic Login
Using Container ID: 102
Using Hostname: omada
Using Disk Size: 8GB
Allocated Cores 2
Allocated Ram 2048
Using Bridge: vmbr0
Using Static IP Address: dhcp
Using Gateway IP Address: Default
Using Apt-Cacher IP Address: Default
Disable IPv6: No
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server Address: Host
Using MAC Address: Default
Using VLAN Tag: Default
Enable Root SSH Access: No
Enable Verbose Mode: No
Creating a Omada LXC using the above default settings
 ✓ Using local for Template Storage.
 ✓ Using local-lvm for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 102 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.178.45 
 ✓ IPv4 Internet Connected
 ✓ DNS Resolved github.com to 140.82.121.4
 ✓ Updated Container OS
 ✓ Installed Dependencies
 ✓ Installed Azul Zulu
 - Installing MongoDB  \
[ERROR] in line 33: exit code 0: while executing command wget -qL http://security.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2.20_amd64.deb
root@prox:~#

**Comment by tteck** [comment]
fixed

**Reply by thealphabeta** [reply]
thanx!  no more issues
